service/src/test/java/uk/gov/hmcts/reform/fpl/service/document/DocumentsListRendererTest.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.reform.fpl.service.document;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.model.documentview.DocumentBundleView;
+import uk.gov.hmcts.reform.fpl.model.documentview.DocumentView;
+import uk.gov.hmcts.reform.fpl.service.CaseUrlService;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.utils.ResourceReader.readString;
+
+@ExtendWith(MockitoExtension.class)
+class DocumentsListRendererTest {
+
```

Comment:
missing edge case when getDocumentUrl() is incorrect and returns  an empty string